yesod-core/Yesod/Core/Class/Yesod.hs

```diff
@@ -366,6 +366,22 @@ sslOnlySessions = (fmap . fmap) secureSessionCookies
     setSecureBit cookie = cookie { setCookieSecure = True }
     secureSessionCookies = customizeSessionCookies setSecureBit
 
+-- | Helps defend against CSRF attacks by setting the SameSite attribute on
+-- session cookies to "Lax".
```

Comment:
> As the spec is still evolving

Hm, ok. Here are some key things that might be worth covering:

* Warn that browser support is recent/forthcoming. Otherwise, people might use SameSite cookies and they'll be confused why it isn't working for them, or it might work for them and they won't realize many of their users aren't protected. A link to caniuse.com would be great, but they don't [support it yet, unfortunately](https://github.com/Fyrd/caniuse/issues/16130).
* Warn that the yesod API may need to change, since the spec is still in flux, if you think that might need to happen
* Suggest checking the latest draft of the spec. Is this URL https://tools.ietf.org/html/draft-west-first-party-cookies sufficient? It takes you the latest version of the spec—not sure if they intend to move it from that site later.